src/Common/src/Interop/Unix/libc/Interop.poll.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class libc
+    {
+        [Flags]
+        internal enum PollFlags : short
+        {
+            POLLIN      = 0x0001,       /* any readable data available */
+            POLLPRI     = 0x0002,       /* OOB/Urgent readable data */
+            POLLOUT     = 0x0004,       /* file descriptor is writeable */
+            POLLRDNORM  = 0x0040,       /* non-OOB/URG data available */
+            POLLWRNORM  = POLLOUT,      /* no write type differentiation */
+            POLLRDBAND  = 0x0080,       /* OOB/Urgent readable data */
+            POLLWRBAND  = 0x0100,       /* OOB/Urgent data can be written */
+            POLLERR     = 0x0008,       /* some poll error occurred */
+            POLLHUP     = 0x0010,       /* file descriptor was "hung up" */
+            POLLNVAL    = 0x0020,       /* requested events "invalid" */
+        }
```

Comment:
The one that I'm using is, the rest are kind of irrelevant for now so I'll remove them. If they are needed down the road, we can adjust then